db/migrate/20201020203844_create_restaurants.rb

```diff
@@ -0,0 +1,17 @@
+class CreateRestaurants < ActiveRecord::Migration[5.2]
+  def up
+    create_table :restaurants, id: false do |t|
+      t.bigint :id, primary: true
+      t.string :name, null: false
+      t.string :cuisine
+      t.float :latitude, null: false
+      t.float :longitude, null: false
+
+      t.timestamps
+    end
+    execute %Q{ ALTER TABLE "restaurants" ADD PRIMARY KEY ("id"); }
```

Comment:
If we are just going to use the `id` column as the primary key, then Rails will do that by default, so we can remove the `id: false` option on line 3, line 4, and line 12.